.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
You can add `.vscode` to `.gitignore` file. By doing so git will stop tracking the vscode config files 

To remove the config files from this commit try this: 

https://stackoverflow.com/questions/12481639/remove-files-from-git-commit

I can also help with this, if it becomes to complicated 😄